src/deepsparse/server/main.py

```diff
@@ -1,352 +0,0 @@
-# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Are we sure that landing that PR will be identical to replacing the old server? Maybe it makes sense to have them both live concurrently (something like server and server_v2) for a while.

Reply:
Yeah good question. I think this is another big reason to have unit/integration tests, otherwise all you can *really* do is try to side-by-side compare things haha. Are there specific cases that you are wondering about?